bucket/sapling.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "20221115.080554.34470671",
+    "description": "A Scalable, User-Friendly Source Control System.",
+    "homepage": "https://sapling-scm.com/",
+    "license": "GPL-2.0-only",
+    "notes": [
+        "The name of the Sapling CLI sl.exe conflicts with the sl shell built-in in PowerShell (sl is an alias for Set-Location, which is equivalent to cd).",
+        "If you want to use sl to run sl.exe in PowerShell, you must reassign the alias.",
+        "You must add the following to your $PROFILE:",
+        "Set-Alias -Name sl -Value '$dir\\sl.exe' -Force -Option Constant,ReadOnly,AllScope"
+    ],
+    "depends": "git",
+    "suggest": {
+        "node.js": [
+            "nodejs",
+            "nodejs-lts"
+        ]
+    },
```

```suggestion
    "suggest": {
        "Node JS": "nodejs-lts",
        "Git": "git"
    },
```
Since Git is not a hard dependency to run the sl.exe CLI, `suggest` is a more flexible option than `depends`